content/docs/latest/installing/vms/proxmoxve.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Running Flatcar Container Linux on Proxmox VE
+linktitle: Running on Proxmox VE
+weight: 30
+---
+
+_While we always welcome community contributions and fixes, please note that Proxmox VE is not an officially supported platform at this time because the release tests don't run for it. (See the [platform overview](/#installing-flatcar).)_
+
+These instructions will walk you through running Flatcar Container Linux on Proxmox VE.
+
+## Choose a channel
+
+Flatcar Container Linux is designed to be updated automatically with different schedules per channel. You can [disable this feature][update-strategies], although we don't recommend it. Read the [release notes][release-notes] for specific features and bug fixes.
+
+Promox VE OEM images are created for both amd64 and arm64 and come in qcow2 compressed format.
+
+How to download a Promox VE qcow2 image file:
+
+```bash
+# THIS IS A DEVELOPMENT BUILD
+# TODO: update link with an alpha build
+wget http://bincache.flatcar-linux.net/images/amd64/9999.0.102+kai-proxmox-support/flatcar_production_proxmoxve_image.img.bz2 | bzip2 -d > flatcar_production_proxmoxve_image.img
+```
+
+## Preparing the image
+
+Promox VE graphical UI only supports ISO images, but the Flatcar image is in QCOW2 format.
+In order to create a VM with our image we'll need to use the command line. Open a shell on the hypervisor, download the image and convert it to RAW :
+
+```bash
+qemu-img convert -f qcow2 -O raw flatcar_production_proxmoxve_image.img flatcar_production_proxmoxve_image.bin
+```
+
+## Creating the VM
+
+Because our image is not in ISO format we'll need to use the command-line to create the VM.
+
+```bash
+export VM_ID=123
+
+# create the vm and import the image to it's disk
+qm create $VM_ID
+qm disk import $VM_ID flatcar_production_proxmoxve_image.bin local
+
+# tell the vm to boot from the imported image
+qm set $VM_ID --scsi0 local:$VM_ID/vm-$VM_ID-disk-0.raw
+qm set $VM_ID --boot order=scsi0
+```
+
+## Configuring the VM with Openstack-style cloud-init config
+
+Our VM can be booted as-is, however we might want to add a cloud-init configuration.
```

Comment:
The VM I booted without config drive is not booting - as it stops at the initrd stage and drops into an emergency shell, because the coreos-metadata service fails.

Reply:
Fixed by:
```
# Create the cloud-init CD-ROM drive which activates the cloud-init options for the VM.
qm set $VM_ID --ide2 local:cloudinit
```